testsuites/CBLTester/System_test_multiple_device/test_system_testing.py

```diff
@@ -0,0 +1,283 @@
+
+            # Deleting docs will affect all dbs as they are synced with SG.
+            for repl_obj, repl, cbl_db, query in zip(replicator_obj_list,
+                                                     replicator_list,
+                                                     cbl_db_list,
+                                                     query_obj_list):
+                t = Thread(target=_replicaton_status_check, args=(repl_obj, repl))
+                t.start()
+                t.join()
+                query.query_get_docs_limit_offset(cbl_db, limit=query_limit, offset=query_offset)
+        _check_doc_count(db_obj_list, cbl_db_list)
+        # removing ids of deleted doc from the list
+        doc_ids = doc_ids - docs_to_delete
+
+        #############################
+        # Creating docs on CBL side #
+        #############################
+        for db_obj, cbl_db, repl_obj, repl, query in zip(db_obj_list,
+                                                         cbl_db_list,
+                                                         replicator_obj_list,
+                                                         replicator_list,
+                                                         query_obj_list):
+            name = db_obj.getName(cbl_db)
+            docs_to_create = ["cbl_{}_{}".format(name, doc_id) for doc_id in range(doc_id_for_new_docs, doc_id_for_new_docs + num_of_docs_to_add)]
+            added_docs = {}
+            new_doc_ids = []
+            for doc_id in docs_to_create:
+                if generator == "complex_doc":
+                    data = complex_doc()
+                elif generator == "four_k":
+                    data = four_k()
+                elif generator == "simple_user":
+                    data = simple_user()
+                else:
+                    data = simple()
+                data["channels"] = channels_sg
+                data["_id"] = doc_id
+                added_docs[doc_id] = data
+                new_doc_ids.append(doc_id)
+            doc_ids.update(new_doc_ids)
+            print "creating {} docs on {} - {}".format(len(docs_to_create),
+                                                       db_obj.getName(cbl_db),
+                                                       new_doc_ids)
+            db_obj.saveDocuments(cbl_db, added_docs)
+            time.sleep(5)
+
+            # Adding docs will affect all dbs as they are synced with SG.
+            t = Thread(target=_replicaton_status_check, args=(repl_obj, repl))
+            t.start()
+            t.join()
+            query.query_get_docs_limit_offset(cbl_db, limit=query_limit, offset=query_offset)
+            time.sleep(5)
+        doc_id_for_new_docs += num_of_docs_to_add
+        _check_doc_count(db_obj_list, cbl_db_list)
+
+        current_time = datetime.now()
+    # stopping replication
+    print "Test completed. Stopping Replicators"
```

Comment:
can you either remove print statements or replace with log_info?

Reply:
changed it to log_info